src/shared/logs-show.c

```diff
@@ -76,7 +76,7 @@ static int print_catalog(FILE *f, sd_journal *j) {
                 return log_oom();
 
         if (colors_enabled())
-                fprintf(f, ANSI_GREY "%s" ANSI_NORMAL " " ANSI_GREEN, prefix);
+                fprintf(f, "%s%s %s%s", ansi_grey(), prefix, ANSI_NORMAL, ansi_green());
```

Comment:
ansi_normal() ?

Reply:
I haven't defined an `ansi_normal()` because it always has the same value, regardless of the color mode.

Reply:
Uh?! `ansi_normal()` is used in other places in this same patch...

Reply:
Sorry, you're right: I wasn't remembering correctly. it is defined, but I haven't used it where there is already a `if (colors_enabled)` or equivalent to save one `get_color_mode` call.